### PR TITLE
Fix arm32 built on amd64 host

### PR DESCRIPTION
**Pull request by nvllsvm**
Make arm Docker build again.

### CODE REVIEW

File: Dockerfile.arm [hunk 1]
```diff
@@ -21,7 +25,9 @@ RUN find . -type d -name obj | xargs -r rm -r
 RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" --self-contained --runtime linux-arm "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
 
 
-FROM debian:buster-slim
+FROM multiarch/qemu-user-static:x86_64-arm as qemu
+FROM arm32v7/debian:buster-slim
+COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y ffmpeg \
```
Comment (Artiume) [inline Dockerfile.arm]: we're not running jellyfin-ffmpeg?

Comment (joshuaboniface) [reply Dockerfile.arm]: Hrm, I had never noticed this. We probably should be.

Comment (nvllsvm) [reply Dockerfile.arm]: No working builds for arm yet.

File: Dockerfile.arm [hunk 2]
```diff
@@ -21,7 +25,9 @@ RUN find . -type d -name obj | xargs -r rm -r
...
  libssl-dev ca-certificates \
```
Comment (Artiume) [inline Dockerfile.arm]:
```suggestion
 && apt-get install --no-install-recommends --no-install-suggests -y jellyfin-ffmpeg
	libssl1.0.0 \
	ca-certificates \
	libfontconfig1 \
	libfreetype6 \
	libomxil-bellagio0 \
	libomxil-bellagio-bin \
	libraspberrypi0 \
```

Comment (Artiume) [reply Dockerfile.arm]: This is what I'm building now, I may have to adjust packages after netcore finishes, I haven't run these packages against debian yet. I still had to add --network-timeout 100000 after I fixed my... issues.

Comment (Artiume) [inline Dockerfile.arm]:
```suggestion
FROM multiarch/qemu-user-static:x86_64-arm as qemu
FROM lsiobase/ubuntu:arm32v7-bionic
COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin

ARG DEBIAN_FRONTEND="noninteractive"
ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn

RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates gnupg curl && \
 curl -s https://repo.jellyfin.org/ubuntu/jellyfin_team.gpg.key | apt-key add - && \
 curl -s https://keyserver.ubuntu.com/pks/lookup?op=get\&search=0x6587ffd6536b8826e88a62547876ae518cbcf2f2 | apt-key add - && \
 echo 'deb [arch=armhf] https://repo.jellyfin.org/ubuntu bionic main' > /etc/apt/sources.list.d/jellyfin.list && \
 echo "deb http://ppa.launchpad.net/ubuntu-raspi2/ppa/ubuntu bionic main">> /etc/apt/sources.list.d/raspbins.list && \
  apt-get install --no-install-recommends --no-install-suggests -y \
 	jellyfin-ffmpeg \
	libssl-1.0.0 \
	libfontconfig1 \
	libfreetype6 \
	libomxil-bellagio0 \
	libomxil-bellagio-bin \
	libraspberrypi0 \
```

Comment (Artiume) [reply Dockerfile.arm]: Don't hate me lol. Feels like hackery, and it should definitely be cleaned up

**Comment by Artiume** [comment]
https://github.com/jellyfin/jellyfin-ffmpeg/issues/17

here's the specific issue that jellyfin-ffmpeg had. On arm, i had 2 dependency issues though.

**Comment by anthonylavado** [comment]
@nvllsvm Any thoughts on @Artiume 's comments?

File: Dockerfile.arm (hunk 2 above)
Comment (nvllsvm) [reply Dockerfile.arm]: If this works, I'm good with it. Can't seem to apply the suggestions though. Open a separate PR.

**Comment by Artiume** [comment]
Ok, go ahead and merge this one. It'll at least verify that we restore arm. I'm not too concerned about my libs just yet, I can do them over the weekend along with the dependency issues for jellyfin-ffmpeg

**Pull request merged**